wandb/apis/public.py

```diff
@@ -4125,9 +4124,14 @@ def __init__(
             extra=entry.extra,
             local_path=entry.local_path,
         )
-        self.name = name
         self._parent_artifact = parent_artifact
 
+    @property
+    def name(self):
+        # TODO(hugh): add telemetry to see if anyone is still using this.
+        wandb.termwarn("ArtifactManifestEntry.name is deprecated, use .path instead")
```

Comment:
Is it okay if I add a term warning and a TODO to actually add telemetry? @szymon-piechowicz-wandb @vwrj

Reply:
lgtm

Reply:
sg